regression/contracts/assigns_enforce_offsets_4/test.desc

```diff
@@ -1,9 +1,17 @@
-KNOWNBUG
+CORE
 main.c
 --enforce-contract foo
+^\[.*\d+\].* line 5 Check validity of assigns clause target 'TRUE: x\[\(.*\)10\]': FAILURE$
+^\[.*\d+\].* line 8 Check that x\[\(.*\)10\] is assignable: SUCCESS$
+^\[.*\d+\].* line 8 dereference failure: pointer outside object bounds in x\[\(.*\)10\]: FAILURE$
 ^EXIT=10$
 ^SIGNAL=0$
 ^VERIFICATION FAILED$
 --
 --
-Check that a write at *x fails when the assigns clause specifies *(x + 1) and the actual underlying object is of size 1.
+Check that a write at *(x+10) fails when the assigns clause specifies *(x + 10)
+and the actual underlying object is too small.
+In that case the target inclusion succeeds because the LHS is in an invalid
+state, but the target validity check and the pointer checks for the assignment
+must fail.
+ 
```

Comment:
Nit pick: lots of whitespace, but no newline at end-of-file.

Reply:
fixed